src/dom/Draggable.js

```diff
@@ -62,6 +62,8 @@ L.Draggable = L.Evented.extend({
 	disable: function () {
 		if (!this._enabled) { return; }
 
+		if (this._lastTarget) { this._onUp({}); }

```

Comment:
See here: https://playground-leaflet.rhcloud.com/vikez/edit?html,output
changed 'dragstart' to 'mousedown' and problem persists.
This is because you set Draggable._dragging on mousedown and you set this._lastTarget on mousemove.
I see a design problem with this. Perhaps set this._lastTarget in mousedown as well?
Or set Draggable._dragging to refer the very Draggable and clear on disable iif Draggable._dragging === this?